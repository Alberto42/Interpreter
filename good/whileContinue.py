i = 0
x = 0
while (x < 10) {
    x = x + 1
    if x * 2 > 5 {
        continue
    }
    i = i + 1
}