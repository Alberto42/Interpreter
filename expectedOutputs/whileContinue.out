Right (OK,fromList [(Ident "i",VInt 2),(Ident "x",VInt 10)])
